app/js/newTest.js

```diff
@@ -0,0 +1,23 @@
+let testForm = document.querySelector('#testForm')
+
+testForm.addEventListener('submit', async function(e) {
+    e.preventDefault()
+    let inputLength = document.querySelector('#testName').value.length
+    if (inputLength > 0 && inputLength < 256) {
+        let testData = {}
+        testData.name = testForm.testName.value
+        testData = await jsonToFormData(testData)
+        const testPath = "test"
+        let response = await sendData(testData, testPath)
+        showConfirmationMessage(response)
+        if (response.success) {
```

Comment:
feels like this if could be in the showConfirmationMessage function? they seem very linked and always needed?

Reply:
no

Reply:
:(

Reply:
dont want it

Reply:
function no longer exists